Apps/frontend/src/views/StartStopButton.svelte

```diff
@@ -6,23 +6,24 @@
 
   const dispatch = createEventDispatcher();
 
-  let hasStarted = true;
+  let hasStarted = false;
   let color = LINE_COLORS_RGBA[channel_id];
   let channelLabel = `CH ${channel_id}`;
 
   const handleStartStop = async () => {
-    hasStarted = !hasStarted;
-    dispatch("startStop", { buttonValue: hasStarted });
+    console.log("erstes hasStarted " + hasStarted);
```

Comment:
Remove logging before merge.
Additionally, please remove unused comments (you added).

Reply:
I hope i got all